pkg/fleet/installer/bootstrap/bootstrap_windows.go

```diff
@@ -51,6 +51,7 @@ func install(ctx context.Context, env *env.Env, url string, experiment bool) err
 
 // downloadInstaller downloads the installer package from the registry and returns the path to the executable.
 func downloadInstaller(ctx context.Context, env *env.Env, url string, tmpDir string) (*iexec.InstallerExec, error) {
+	var installPath string
```

Comment:
nit: you shouldn't need to define it up here

```suggestion
	var installPath string
```